### ISSUE TITLE
Use UriComponentsBuilder.clone() instead of workaround

### ISSUE THREAD

**Issue by steve-todorov** [strongbox/strongbox#1731]
# Task Description

This is a follow-up task to switch back to using `UriComponentsBuilder.clone()` instead of a workaround we had to introduce as part of #1704 since we were hitting spring-projects/spring-framework#24772.

# Task Relationships

This task:
* Depends on: `spring-web@5.2.6` being released at some point in early May 2020 (maybe?).
* Is a follow-up of: #1704 (PR #1708)
* Relates to: spring-projects/spring-framework#24772

# Help

* [Our chat](https://chat.carlspring.org/)
* Points of contact:
  * @steve-todorov

**Comment by parameswaranmarimuthu** [comment]
@steve-todorov , I would like to start on this. Could you please assign this to me?

**Comment by steve-todorov** [comment]
@parameswaranmarimuthu hi and thanks for your interest in this issue. However it's still blocked (as the labels show). Maybe check another issue?

**Comment by AdrianMirica** [comment]
Hello, the issue is still blocked ? If no, maybe I can help with it.
